### ISSUE TITLE
Some issues with different menus under Career

### ISSUE THREAD

**Issue by LennardF1989** [thepeacockproject/Peacock#228]
Compared Peacock to Official in excruciating detail. most likely also counts for other locations in the menu.

# Career > Statistics

## Season 1:
Escalation Track 1/2 should not show for The Icon, Landslide and The Author
- It doesn't for "The Source", "The Vector" and "Patient Zero" already, weirdly enough.

The Icon and Landslide are swapped position wise.

# Career > Challenges
Freeform Training and The Final Test should move to Season 1

# In advance for Arcade:
Arcade challenges show on:
- The Showstopper
- World of Tomorrow
- The Icon
- A Gilded Cage
- A House Built on Sand
- Club 27
- Freedom Fighters
- Situs Inversus
- Nightall
- Three-Headed Serpent
- Another Life
- The Ark Society
- On Top of the World
- Death in the Family
- Apex PRedator
- End of an Era
- The Farewell

# Other
Random thought: Why do Escalation tracks show on locations that are not applicable (also on Official)

**Comment by moonysolari** [comment]
> The Icon and Landslide are swapped position wise.

Opened #236 to fix it.

> Freeform Training and The Final Test should move to Season 1

They are in Season 3 on official.

**Comment by AnthonyFuller** [comment]
Do you think we should move it to Season 1? My rational behind this is because now with a purchase of Hitman WoA, you have all Hitman 2016 locations. ICA Facility was only in Season 3 because of the free starter pack (and people possibly not having 2016 maps on purchase).

**Comment by LennardF1989** [comment]
This is not correct @moonysolari - I based this on screenshots I made of official:

![Hitman 3 Screenshot 2023 04 19 - 00 05 00 59](https://user-images.githubusercontent.com/869973/233700709-36254579-6975-450c-b543-3fa0a10aaf53.png)

**Comment by AnthonyFuller** [comment]
They are in Season 3 on other places in the menu, challenges, mastery, contract creation, etc.

**Comment by moonysolari** [comment]
> I based this on screenshots I made of official:

You originally wrote that bullet point under the Career > Challenges headline, and they are indeed in HITMAN 3 on official in that menu. As for the statistics menu, I believe it has been fixed. I have also checked the newly added Arcade challenges and those locations are indeed exactly where they appear on. Therefore I believe all problems raised by this issue have been addressed and it can be closed.
